### ISSUE TITLE
from-issue-210--optimize-projection-dims-cerebros-params-and-train-set-size

### ISSUE THREAD

**Issue by david-thrower** [david-thrower/cerebros-core-algorithm-alpha#211]
Take the proof of concept on autoregressive generation in #210  using Cerebros with a single set of logits of shape (vocabulary_size) and optimize the tunable params and training set size.

**Comment by david-thrower** [comment]
As a comment this was successful. Added projection layers to compress the embedding output.
